### DISCUSSION TITLE
Welcome to Harvest Monster Discussions!

### DISCUSSION THREAD

**Discussion by ryanbarr** [ryanbarr/harvest-monster#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by floki1234** [comment]
I always write my crafts in by hand because the OCR is extremely slow. But the issue with this is that the prices don't autoupdate. I have to go to Settings and then toggle prices off and on.

Could you place a button on the main page that is "Autoupdate prices", which does this? It would save me a lot of hassle, and I think others would appreciate this too.

**Reply by ryanbarr** [reply]
Hey @floki1234! Thanks for the suggestion, a few notes:

1. I've created a new issue to allow you to update prices from the crafts page to help make yours (and others) experience a little more user-friendly. We'll get this added in the next update! You can follow the issue here: #53 
2. If you have specific screenshots that are slow when using OCR, posting them as [a new issue](https://github.com/ryanbarr/harvest-monster/issues/new/choose) would be incredibly helpful in solving performance concerns.
3. We have planned updates for OCR and matching that will come in a future update that is currently targeted for prior to Path of Exile 3.19.
